testing/marketplace/deployer/helm_tiller_onbuild/helm-dependency-build/schema.yaml

```diff
@@ -0,0 +1,34 @@
+properties:
+application_api_version: v1beta1
+properties:
+  name:
+    type: string
+    x-google-marketplace:
+      type: NAME
+  namespace:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+  application_uid:
+    type: string
+    x-google-marketplace:
+      type: APPLICATION_UID
+  marketplace-integration.image:
+    type: string
+    x-google-marketplace:
+      type: IMAGE
+  marketplace-integration.service_account:
+    type: string
+    x-google-marketplace:
+      type: SERVICE_ACCOUNT
+      serviceAccount:
+        roles:
+        - type: Role
+          rulesType: PREDEFINED
+          rulesFromRoleName: cluster-admin
+required:
+- name
+- namespace
+- application_uid
+- marketplace-integration.image
+- marketplace-integration.service_account
```

Comment:
I forgot--why does this need a service account again? I thought we're creating them directly in the hooks?